test/timing_load_creds.c

```diff
@@ -12,14 +12,15 @@
 
 #include <openssl/e_os2.h>
 
-#ifdef OPENSSL_SYS_UNIX
+#if defined OPENSSL_SYS_UNIX
```

Comment:
We're normally put () around the `OPENSSL_SYS_UNIX`.
Or, better, don't change this?

Reply:
Yeah